src/Microsoft.DotNet.Build.Tasks/PackageFiles/crossgen.sh

```diff
@@ -77,6 +77,7 @@ fi
 
 __MyGetFeed=${BUILDTOOLS_CROSSGEN_FEED:-https://dotnet.myget.org/F/dotnet-core/api/v3/index.json}
 __targetDir=$1
+__packageRid=$2
```

Comment:
This causes an error when only one parameter is passed: `$2` is unbound and this script has `set -euo pipefail`. I think you need `${2:-}`.

/cc @safern

Reply:
@dagood, I missed pipefaul and I didn’t knew its function, will send a fix. Sorry for the troubles. :(

Reply:
Thanks! FWIW, it's the `u` that's the direct cause. (That command is equivalent to `set -e -u -o pipefail`.) http://wiki.bash-hackers.org/commands/builtin/set has some info about this and I've been finding that wiki a pretty good reference in general.

Reply:
I can add the fix to my PR itself if that is quicker.

Reply:
Sorry, I didn't read the location of the file affected :neutral_face:, I thought it was in this repo. Anyway I can make one to fix this :smile:

Reply:
I see--just to clarify then: this thread is on a BuildTools PR, so it is this repo. 😄

I think it's up to you and @am11 who should make the PR. Since it's so small, maybe just whoever gets to it first.

Reply:
> I see--just to clarify then: this thread is on a BuildTools PR, so it is this repo. 😄

haha I know, just miss-reading and commenting without looking carefully at where/what was affected 😆